src/models/github-server-app.ts

```diff
@@ -70,6 +70,26 @@ export class GitHubServerApp extends EncryptedModel {
 		});
 	}
 
+	/**
+	 * Get all GitHubServerApps with installationId
+	 *
+	 * @param {{installationId: number}} installationId
+	 * @returns {GitHubServerApp[]}
+	 */
+	static async getForInstallationId(
```

Comment:
To stick with nomenclature of sequelize, we should use `find` not `get`

Reply:
Updated the name to `findForInstallationId`